content/kubeone/master/guides/addons/_index.en.md

```diff
@@ -81,9 +85,105 @@ the path is relative to the KubeOne configuration file. This means that
 
 {{% notice note %}}
 Subdirectories are not considered when applying addons. Only addons in the root
-of the provided directory will be applied.
+of the provided directory will be applied. E.g. `./addons/addon_name1` is
+supported addon while `./addons/groupping/addon_name1` and
+`./addons/addon_name1/subdir1` does not.
```

Comment:
This is a bit similar to the previous notice, so it might be confusing. What do you think about something like this:

> Addons can be organized into subdirectories, but only one level of subdirectories is supported. For example, `./addons/example-addon-1` is supported and YAML manifest in that directory will be deployed, but `./addons/example-addon-1/subdirectory-2` directory will be entirely ignored.